<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>KisWdgAutobrush</class>
  <widget class="QWidget" name="KisWdgAutobrush" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>373</width>
        <height>200</height>
      </rect>
    </property>
    <property name="sizePolicy" >
      <sizepolicy>
        <hsizetype>1</hsizetype>
        <vsizetype>1</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
    <layout class="QGridLayout" >
      <item row="0" column="1" >
        <layout class="QVBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <item>
            <widget class="Q3GroupBox" name="grpSize" >
              <property name="title" >
                <string>&amp;Size</string>
              </property>
              <layout class="QHBoxLayout" >
                <property name="spacing" >
                  <number>0</number>
                </property>
                <item>
                  <layout class="QGridLayout" >
                    <property name="spacing" >
                      <number>0</number>
                    </property>
                    <property name="margin" >
                      <number>0</number>
                    </property>
                    <item row="0" column="0" >
                      <widget class="QLabel" name="textLabel2" >
                        <property name="sizePolicy" >
                          <sizepolicy>
                            <hsizetype>1</hsizetype>
                            <vsizetype>1</vsizetype>
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                          </sizepolicy>
                        </property>
                        <property name="text" >
                          <string/>
                        </property>
                        <property name="pixmap" >
                          <pixmap>image0</pixmap>
                        </property>
                      </widget>
                    </item>
                    <item row="0" column="1" >
                      <widget class="QSpinBox" name="spinBoxWidth" >
                        <property name="focusPolicy" >
                          <enum>Qt::ClickFocus</enum>
                        </property>
                        <property name="maximum" >
                          <number>9999</number>
                        </property>
                        <property name="value" >
                          <number>5</number>
                        </property>
                      </widget>
                    </item>
                    <item row="1" column="0" >
                      <widget class="QLabel" name="textLabel2_2" >
                        <property name="text" >
                          <string/>
                        </property>
                        <property name="pixmap" >
                          <pixmap>image1</pixmap>
                        </property>
                      </widget>
                    </item>
                    <item row="1" column="1" >
                      <widget class="QSpinBox" name="spinBoxHeigth" >
                        <property name="focusPolicy" >
                          <enum>Qt::ClickFocus</enum>
                        </property>
                        <property name="maximum" >
                          <number>9999</number>
                        </property>
                        <property name="value" >
                          <number>5</number>
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
                <item>
                  <widget class="QToolButton" name="bnLinkSize" >
                    <property name="sizePolicy" >
                      <sizepolicy>
                        <hsizetype>1</hsizetype>
                        <vsizetype>0</vsizetype>
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                    <property name="minimumSize" >
                      <size>
                        <width>16</width>
                        <height>0</height>
                      </size>
                    </property>
                    <property name="maximumSize" >
                      <size>
                        <width>16</width>
                        <height>32767</height>
                      </size>
                    </property>
                    <property name="text" >
                      <string/>
                    </property>
                    <property name="checkable" >
                      <bool>true</bool>
                    </property>
                  </widget>
                </item>
              </layout>
            </widget>
          </item>
          <item>
            <widget class="Q3GroupBox" name="grpFade" >
              <property name="title" >
                <string>&amp;Fade</string>
              </property>
              <layout class="QHBoxLayout" >
                <property name="spacing" >
                  <number>0</number>
                </property>
                <item>
                  <layout class="QGridLayout" >
                    <property name="spacing" >
                      <number>0</number>
                    </property>
                    <property name="margin" >
                      <number>0</number>
                    </property>
                    <item row="0" column="0" >
                      <widget class="QLabel" name="textLabelHorizontal" >
                        <property name="text" >
                          <string/>
                        </property>
                        <property name="pixmap" >
                          <pixmap>image0</pixmap>
                        </property>
                      </widget>
                    </item>
                    <item row="0" column="1" >
                      <widget class="QSpinBox" name="spinBoxHorizontal" >
                        <property name="focusPolicy" >
                          <enum>Qt::ClickFocus</enum>
                        </property>
                        <property name="maximum" >
                          <number>9999</number>
                        </property>
                        <property name="value" >
                          <number>1</number>
                        </property>
                      </widget>
                    </item>
                    <item row="1" column="0" >
                      <widget class="QLabel" name="textLabel2_2_2" >
                        <property name="text" >
                          <string/>
                        </property>
                        <property name="pixmap" >
                          <pixmap>image1</pixmap>
                        </property>
                      </widget>
                    </item>
                    <item row="1" column="1" >
                      <widget class="QSpinBox" name="spinBoxVertical" >
                        <property name="focusPolicy" >
                          <enum>Qt::ClickFocus</enum>
                        </property>
                        <property name="maximum" >
                          <number>9999</number>
                        </property>
                        <property name="value" >
                          <number>1</number>
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
                <item>
                  <widget class="QToolButton" name="bnLinkFade" >
                    <property name="sizePolicy" >
                      <sizepolicy>
                        <hsizetype>1</hsizetype>
                        <vsizetype>0</vsizetype>
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                    <property name="minimumSize" >
                      <size>
                        <width>16</width>
                        <height>0</height>
                      </size>
                    </property>
                    <property name="maximumSize" >
                      <size>
                        <width>16</width>
                        <height>32767</height>
                      </size>
                    </property>
                    <property name="text" >
                      <string/>
                    </property>
                    <property name="checkable" >
                      <bool>true</bool>
                    </property>
                  </widget>
                </item>
              </layout>
            </widget>
          </item>
        </layout>
      </item>
      <item row="0" column="0" >
        <layout class="QVBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <item>
            <widget class="QToolButton" name="brushPreview" >
              <property name="sizePolicy" >
                <sizepolicy>
                  <hsizetype>7</hsizetype>
                  <vsizetype>7</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="minimumSize" >
                <size>
                  <width>95</width>
                  <height>95</height>
                </size>
              </property>
              <property name="text" >
                <string/>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QComboBox" name="comboBoxShape" >
              <property name="focusPolicy" >
                <enum>Qt::ClickFocus</enum>
              </property>
              <item>
                <property name="text" >
                  <string>Circle</string>
                </property>
              </item>
              <item>
                <property name="text" >
                  <string>Square</string>
                </property>
              </item>
            </widget>
          </item>
          <item>
            <spacer name="spacer4" >
              <property name="sizeHint" >
                <size>
                  <width>20</width>
                  <height>0</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Qt::Vertical</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction></pixmapfunction>
  <images>
    <image name="image0" >
      <data format="PNG" length="236" >89504e470d0a1a0a0000000d49484452000000190000001a0806000000427df7cd000000b3494441544889dd963112c420084581d97acdfd4f29b9005b3913135050d722bf54f0018223e6ccf06f7db4c594be4d27e63304a190f525885e20154433f646ea050d6512d516887af10075c922f5d7e4ca24da4d439059f803721c4946a1962f798c6641d4da5c05421101445c06b82a6746804d73822202cc67956289a0c89a937b77596790b638abfb59d4da5c017840664196aff97679deab62d39bfcf73cf5ef8160ef4bb4e2e7627657e4909eb694eb071e3b4da9ba4f1ddb0000000049454e44ae426082</data>
    </image>
    <image name="image1" >
      <data format="PNG" length="242" >89504e470d0a1a0a0000000d49484452000000190000001a0806000000427df7cd000000b9494441544889dd95410ec4200845b1e9ba78ff538a176056260c5181569bccfca590ff2a204da510ecd601008078ed8778953372cec8db20d23c0a72417aa611900999997941a795500a256d2acf7a434354bf212d4927ebc488a417518d4dd75d0dcbb5f2ed983d59f101af94eb7f20d39e3c1963136299f7e2b361f89d9e8c6ed16efc4ee3892a205ecb9adc24fdcc176ffd4b9839e9b896592eb9d623b1106464e605b821da340200086e61afb91ea2a377e89577e57f00357f504513c00b590000000049454e44ae426082</data>
    </image>
  </images>
</ui>
